opentracing-spring-cloud/src/main/java/io/opentracing/contrib/spring/cloud/jdbc/JdbcAspect.java

```diff
@@ -0,0 +1,38 @@
+package io.opentracing.contrib.spring.cloud.jdbc;
+
+import io.opentracing.contrib.jdbc.TracingConnection;
+import org.aspectj.lang.ProceedingJoinPoint;
+import org.aspectj.lang.annotation.Around;
+import org.aspectj.lang.annotation.Aspect;
+
+import javax.sql.DataSource;
+import java.sql.Connection;
+
+/**
+ * Spring AOP Aspect wrapping JDBC-related calls, delegating as much as possible to the official OpenTracing Java JDBC
+ * framework integration.
+ *
+ * @author Juraci Paixão Kröhling
+ */
+@Aspect
+public class JdbcAspect {
```

Comment:
Is this aspect loaded even though it is not in auto-config?

Reply:
It is, at least on the tests :)

Reply:
it works there because the tests is directly configured to use this aspect